plugins/catalog-backend/README.md

```diff
@@ -87,6 +87,25 @@ yarn start
 This will launch both frontend and backend in the same window, populated with
 some example entities.
 
+## Audit Events
+
+- **`CatalogEntityAncestryFetch`**: Tracks `GET` requests to the `/entities/by-name/:kind/:namespace/:name/ancestry` endpoint which return the ancestry of an entity.
+- **`CatalogEntityBatchFetch`**: Tracks `POST` requests to the `/entities/by-refs` endpoint which return a batch of entities.
+- **`CatalogEntityDelete`**: Tracks `DELETE` requests to the `/entities/by-uid/:uid` endpoint which delete an entity. Note: this will not be a permanent deletion and the entity will be restored if the parent location is still present in the catalog.
```

Comment:
Just to try out some potential patterns here:

Current:
```
CatalogEntityAncestryFetch
CatalogEntityBatchFetch
CatalogEntityDelete
```

kebab dot:
```
catalog.entity.ancestry-fetch
catalog.entity.batch-fetch
catalog.entity.delete
```

camel dot:
```
catalog.entity.ancestryFetch
catalog.entity.batchFetch
catalog.entity.delete
```

Flat camel dot:
```
catalog.entityAncestryFetch
catalog.entityBatchFetch
catalog.entityDelete
```

Flat camel but with a module in the mix:
```
catalog.entityAncestryFetch
catalog.entityBatchFetch
catalog.githubProvider.someMadeUpAction
```

Flat kebab but with a module in the mix:
```
catalog.entity-ancestry-fetch
catalog.entity-batch-fetch
catalog.github-provider.some-made-up-action
```

Kebab but with an explicit module in the mix:
```
catalog.entity.ancestry-fetch
catalog.entity.batch-fetch
catalog.module.github-provider.some-made-up-action
```

Kebab but with an explicit plugin and module in the mix:
```
plugin.catalog.entity.ancestry-fetch
plugin.catalog.entity.batch-fetch
plugin.catalog.module.github-provider.some-made-up-action
```

I realized while typing out this last one that we should probably have the framework provide the plugin ID, the auditor service is plugin scoped after all. That kinda points us in the direction of using the same format as we use for plugin IDs overall, with some delimiter that doesn't conflict with that. Since plugin IDs use kebab-case I'd say we should go with that: https://backstage.io/docs/backend-system/architecture/naming-patterns#plugins + dot-separation.

Reply:
Upon further reflection, it seems unnecessary to mandate the inclusion of the plugin ID in the naming scheme. Since the `.child()` method already associates the event with the corresponding plugin, the ID is implicitly captured. This simplifies the naming process while retaining the crucial link between events and their originating plugins. Thoughts?